learning_journal/views/default.py

```diff
@@ -0,0 +1,67 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound
+from ..models import Entry
+
+
+@view_config(
+    route_name='home', renderer='learning_journal:templates/home.html'
+)
+def home_view(request):
+    """Render home.html at '/'"""
+    journal_entries = request.dbsession.query(Entry).order_by(
+            Entry.creation_date.desc()
+    ).all()
```

Comment:
This is a little hard to read.  I would consider setting up the query on one line, and then filtering it on the next:

```python
query = request.dbsession.query(Entry)
journal_entries = query.order_by(Entry.creation_date.desc()).all()
```